src/vuefire.js

```diff
@@ -198,6 +198,7 @@ function ensureRefs (vm) {
 var init = function () {
   var bindings = this.$options.firebase
   if (!bindings) return
```

Comment:
I think it makes sense to move this one line below, so if the function is returning nothing we can skip too